src/features/canvass/components/CanvassSidebar/index.tsx

```diff
@@ -53,7 +53,19 @@ const CanvassSidebar: FC<Props> = ({ assignment }) => {
             />
             <Divider sx={(theme) => ({ bgcolor: theme.palette.grey[100] })} />
             <Typography sx={{ pb: 2, pt: 2 }} variant="body2">
-              <ZUIMarkdown markdown={assignment.instructions} />
+              <ZUIMarkdown
+                BoxProps={{
+                  ref: (elem: HTMLElement) => {
+                    if (elem) {
+                      const aNodes = elem.querySelectorAll('a');
+                      aNodes.forEach((node) => {
+                        node.setAttribute('target', '_blank');
+                      });
+                    }
+                  },
+                }}
```

Comment:
This could potentially become a property on `ZUIMarkdown`, e.g. `forceTargetBlank`, and could also be applied to the instructions in the landing page before the map. I think this could be a nice entry-level issue.

Reply:
Made into issue #2806